user-interface/6004-CHAR-chart.md

```diff
@@ -0,0 +1,106 @@
+# Chart
+
+## Display options
+- **Must** be able to change time interval from a list of intervals ([1003-TRAN-001](#1003-TRAN-001)) :    
+    - 1m  
+    - 5m  
+    - 15m     
+    - 1H  
+    - 6H  
+    - 1D
+    
+- **Must** be able to change display type from a list of display types ([1003-TRAN-001](#1003-TRAN-001)):   
+    - Moutain     
+    - Candlestick     
+    - Line    
+    - OHLC    
+
+- **Must** icon will change depending on the display type selected([1003-TRAN-001](#1003-TRAN-001)):   
+    - Moutain     
+    - Candlestick     
+    - Line    
+    - OHLC   
+
+- **Must** be able to add overlays by selecting from a list of overlays ([1003-TRAN-001](#1003-TRAN-001)):  
+    - Bollinger bands     
+    - Envelope    
+    - EMA     
+    - Moving Average  
+    - Price monitoring bounds     
+
+- **Must** be able to add studies by selecting from a list of studies ([1003-TRAN-001](#1003-TRAN-001)):  
+    - Eldar-ray   
+    - Force index     
+    - MACD    
+    - RSI     
+    - Volume   
+
+    
+- **Must** be able to add multiple studies at the same time ([1003-TRAN-001](#1003-TRAN-001))
+
+- **Must** be able to close any studies selected ([1003-TRAN-001](#1003-TRAN-001))
+
+- **Must** be able to add multiple overlays at the same time ([1003-TRAN-001](#1003-TRAN-001))
+
+- **Must** be able to close any overlays selected ([1003-TRAN-001](#1003-TRAN-001))
+
+## Price and Time
+
+- **Must** see details of price from where my mouse cursor is on the chart([1003-TRAN-001](#1003-TRAN-001)):  
```

Comment:
We might have to leave any ACs that cover the actual chart rendering untested for now as we'll need some additional tooling due to it being rendered with canvas

Reply:
Agree with the tooling bit, But I think we still want ACs on the behaviours in the spec? So that we know in our coverage what coverage we actually have.